internal/provider/resources/install/okta/group_assignment_staged.go

```diff
@@ -0,0 +1,115 @@
+package installokta
+
```

Comment:
Is this necessary? I don't see any changes to Okta that we're making using this resource's data.